playbot/bot.py

```diff
@@ -37,21 +38,25 @@ def on_welcome(self, c, e):
             time.sleep(0.5)
 
     def on_privmsg(self, c, e):
-        self.do_command(e, e.arguments[0], *e.arguments[1:])
+        e.target = re.sub("!.*","",e.source)
+        e.source = self.nickname
```

Comment:
sorry to be so fussy, but I'm not sure why we are setting e.source to our own nick? it should be the message source?